aten/src/ATen/quantized/Quantizer.cpp

```diff
@@ -159,6 +162,21 @@ Tensor dequantize_tensor(Tensor qtensor, Tensor rtensor, double scale, int64_t z
 }
 #else  // USE_FBGEMM
 
+#if defined(__ANDROID__) && !defined(__NDK_MAJOR__)
+template <class T>
+inline float Round(const float x) {
+  return ::nearbyintf(x);
```

Comment:
Are you sure you want `nearbyint` rather than `round`?  The latter ignores rounding mode flags and is usually more performant.

Reply:
Thanks for pointing it out, I was trying to be more consistent with what we're already using today in caffe2 and fbgemm::Quantize (https://github.com/pytorch/FBGEMM/blob/7f55d6c14fb8ff2b0b03ddf9c4166bd052460fec/include/fbgemm/QuantUtils.h#L65)